kevm-pyk/src/kevm_pyk/kproj/evm-semantics/gas.md

```diff
@@ -73,9 +73,9 @@ module INFINITE-GAS
     rule #gas(G) /Gas #gas(G') => #gas(G /Int G')  requires G' =/=Int 0
 
     rule _:Int    <Gas #gas(_) => true
-    rule #gas(_)  <Gas _       => false [simplification]
```

Comment:
@ehildenb  Is there a reason to keep these simplifications? Removing them shouldn't affect the performance on kontrol since gas computations are turned off by default.